python/GafferUI/VectorDataWidget.py

```diff
@@ -905,11 +919,18 @@ def flags( self, index ) :
 
 	def data( self, index, role ) :
 
+		column = self.__columns[index.column()]
+		if role == QtCore.Qt.BackgroundColorRole :
+
+			if column.index % 2 == 0:
```

Comment:
I'm bikeshedding a bit here, but could these match the colours in the PathListingWidget (HierarchyView). In this context the lighter colour feels like subtle highlighting to me, rather than an alternate background. Maybe we should also update the SceneInspector to match?

Reply:
How about updating _Stylesheet to allow querying the parameters which are used to build the stylesheet colours?

Reply:
What I'd really like to do longer term is find a way of unifying the `GafferUI::Style` stuff we use for the Gadgets with the stylesheet stuff we use for the Widgets. That's not going to happen now though. Are you thinking of just exposing variables like `_Stylesheet._highlightColor` and so on? As long is it all remains protected that sounds good to me.